examples/kubernetes/dynamic_provisioning_s3/README.md

```diff
@@ -45,9 +45,9 @@ Update `spec.resource.requests.storage` with the storage capacity to request. Th
 ### Deploy the Application
 Create PVC, storageclass and the pod that consumes the PV:
 ```sh
->> kubectl apply -f examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml
->> kubectl apply -f examples/kubernetes/dynamic_provisioning/specs/claim.yaml
->> kubectl apply -f examples/kubernetes/dynamic_provisioning/specs/pod.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning_s3/specs/storageclass.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning/specs_s3/claim.yaml
```

Comment:
Should be 
```suggestion
>> kubectl apply -f examples/kubernetes/dynamic_provisioning_s3/specs/claim.yaml
```

Reply:
Could you fix the path here to be `kubectl apply -f examples/kubernetes/dynamic_provisioning_s3/specs/claim.yaml` ? then the PR is good to merge.